enterprise/models.py

```diff
@@ -169,6 +178,28 @@ def requests_data_sharing_consent(self):
         """
         return self.enable_data_sharing_consent and self.enforce_data_sharing_consent != self.EXTERNALLY_MANAGED
 
+    def get_course_enrollment_url(self, course_run_key):
+        """
+        Return enterprise landing page url for the given course.
+
+        Arguments:
+            course_run_key (str): The course run id for the course to be displayed.
+        Returns:
+            (str): Enterprise landing page url.
+        """
+        if configuration_helpers is None:
+            raise NotConnectedToOpenEdX(
+                _("This package must be installed in an EdX environment to look up configuration.")
+            )
+
+        return urljoin(
+            configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
```

Comment:
Actually, since we're using `configuration_helpers`, the above comment is somewhat irrelevant, since this codepath _can't_ operate without `edx-platform`, but I'm gonna leave it for posterity anyway.